packages/test-helpers/src/asserts/assertEvent.js

```diff
@@ -1,7 +1,17 @@
 const { assert } = require('chai')
 const { isAddress, isBN, toChecksumAddress } = require('web3-utils')
 
-const { getEventAt, getEvents } = require('./events')
+const { getEvents, getEventAt } = require('../')
+
+function normalizeArg(arg) {
+  if (isBN(arg)) {
```

Comment:
There were some versions of `web3-utils` that do not include `isBN`.

Reply:
This is likely from the 0.2 series—I think since the 1.0 series started, and they moved over the `bn.js`, this has existed.